common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -534,6 +534,8 @@
 
                 .speed-option,
                 .control-lang {
+                  @include border-left($baseline/10 solid rgb(14, 166, 236));
```

Comment:
Thanks @muzaffaryousaf that's fine. I'll have someone here review the styling. Focus on the video code itself.